.changeset/lazy-dolls-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ant-design/web3-icons': patch
+---
+
+Fix the correct name for the Cronos icon, update the name for the token, and add a new token icons category.
```

Comment:
```suggestion
fix: Fix the correct name for the Cronos icon, update the name for the token, and add a new token icons category.
```

Reply:
Thank you for the suggestion. The wording has been adjusted accordingly.